client/signup/steps/design-type-with-store/pressable-store/index.jsx

```diff
@@ -67,7 +68,9 @@ export default React.createClass( {
 			return;
 		}
 
-		window.location.href = `https://my.pressable.com/signup/ecommerce-five-sites?email=${ encodeURIComponent( this.state.email ) }&utm_source=wordpresscom&utm_medium=signupref&utm_campaign=wpcomecomm2`;
+		analytics.tracks.recordEvent( 'calypso_triforce_partner_redirect', { partner_name: 'Pressable' } );
```

Comment:
We definitely want to replace this analytics call here with the appropriate Redux action as in the other components.

Generally speaking, we should never call `lib/analytics` in any React component. In almost as general of a sense, we should avoid referencing `window` and `document` as much as possible; maybe it's possible to never reference those.